commitlint.config.js

```diff
@@ -1 +1,25 @@
-module.exports = { extends: ['@commitlint/config-conventional'] };
+module.exports = {
+    extends: ['@commitlint/config-conventional'],
+
+    rules: {
+        'body-max-line-length': [1, 'always', 80],
+        'header-max-length': [1, 'always', 72],
```

Comment:
Why would body and header be different length? config-conventional matches them both at 100.

Reply:
For header it's quite obvious: it's recommended here:
* https://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html
* https://cbea.ms/git-commit/#wrap-72

and also GitHub trims header at 72.

For body I looked at staging vim vertical line which is at 80 chars.

Reply:
Main problem is I've often (enough) found "longer === better", to fit "close <#x>, fixup <id>" statements etc.

I think we should just leave this to upstream default, and not try to go back to stone age terminal widths.

Reply:
OK. I removed body max length (upstream is 100) but kept header as warning.

I think soft limit is good as you need to think about message being consistent, from the other side it does not block you when it's impossible and message needs to be longer